### PR TITLE
[15.0][MIG] l10n_th_gov_purchase_request

### PR DESCRIPTION
**Pull request by ps-tubtim**
- Add purchase exception
- Not allow edit purchase request number
- Allow edit purchase representative in the state to approve and approve

Waiting for

- [x] https://github.com/OCA/purchase-workflow/pull/1637
- [x] https://github.com/OCA/purchase-workflow/pull/1638
- [x] https://github.com/OCA/purchase-workflow/pull/1639

### CODE REVIEW

**Comment by Taewii23** [comment]
Create purchase agreement from PR/PR line,
As-is: select 3 PR lines --> generate 3 Purchase Agreements
Should be: select many PR lines --> generate 1 Purchase Agreement

Selection_010
Selection_011

File: l10n_th_gov_purchase_request/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+This module add following features to Purchase Rquest document, aimed to follow Thai Government procurement regulation.
```
Comment (Taewii23) [inline l10n_th_gov_purchase_request/readme/DESCRIPTION.rst]:
```suggestion
This module add following features to Purchase Request document, aimed to follow Thai Government procurement regulation.
```

**Comment by Saran440** [comment]
@Taewii23 Fixed work typo and agreement create following line is bug. i fix it in https://github.com/OCA/purchase-workflow/pull/1639

Thank you for your review.

**Review by Taewii23 — APPROVED** [review]
ok

**Review by Saran440 — APPROVED** [review]
Code Review :+1:

**Pull request merged**